deep_qa/run.py

```diff
@@ -0,0 +1,160 @@
+from typing import Union
+import sys
+import logging
+import shutil
+
+import random
+import pyhocon
+import numpy
+
+# pylint: disable=wrong-import-position
+from deep_qa.common.params import Params, replace_none, ConfigurationError
```

Comment:
This should be `from .common.params import ...`, and similarly on the next line.